### PR TITLE
Sort the list of trading names on the firm index page.

### PR DESCRIPTION
**Pull request by munckymagik**

### CODE REVIEW

**Comment by rnalexander** [comment]
:+1:

File: Gemfile [hunk 1]
```diff
@@ -24,7 +24,9 @@ gem 'dough-ruby',
 gem 'jquery-rails'
 gem 'kaminari'
 gem 'letter_opener', group: :development
-gem 'mas-rad_core', '0.0.55'
+gem 'mas-rad_core',
+    github: 'moneyadviceservice/mas-rad_core',
+    branch: 'firm_sorting_scopes'
```
Comment (tomas-stefano) [inline Gemfile]: Is ok to use this branch?

**Comment by tomas-stefano** [comment]
So :+1:

**Comment by munckymagik** [comment]
Apologies @rnalexander and @tomas-stefano. I wasn't sure how best to do this, but I'm hoping this is the simplest way:

I've added an additional commit to this PR which is essentially a fix for the issue we were trying to address in PR #212.

The new seed which makes this reproducible is `bundle exec rspec --seed 22050`.

UPDATE a new random failure has occurred. Investigating (fire-fighting) :(

**Comment by munckymagik** [comment]
Ok so we've finally got a Travis pass. Apologies about the noise, but would you guys mind giving your thoughts (or thumbs) on the final 2 commits?

If any of this doesn't make sense, I'm in the office tomorrow (MAS) so please do come for a chat :)

**Comment by rnalexander** [comment]
Like it.  Making the sort obvious is a good way to prevent the [0]<->.first flopping problem. 
:+1:

**Comment by tomas-stefano** [comment]
Looks good :+1:

**Comment by munckymagik** [comment]
Great thanks guys :)

**Pull request merged**